packages/documentation/src/pages/when-to-use-usfs.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: When to use the US Forms System
+---
+
+## General structure of applications built with US Forms System
+
+The US Forms System provides a simple way to create a consistent form experience for veterans. Applications created with this framework follow this flow:
```

Comment:
Not really sure how this document fits into the overall flow - we could expand the explanation to include more overview info or include this in another overview doc.

Reply:
I think this probably would fit into an overview/introduction page.